index.html

```diff
@@ -0,0 +1,443 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Teardown Project</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-logo">
+            <h3>site logo</h3>
+        </div>
+        <div class="header-items">
+            <div class="header-images">
+
+            </div>
+            <h2>
+                Articles
+            </h2>
+            <h3>
+                Lorem ipsum
+            </h3>
+        </div>
+        <div class="header-items">
+            <div class="header-images">
+
+            </div>
+            <h2>
+                Books
+            </h2>
+            <h3>
+                Lorem ipsum
+            </h3>
+        </div>
+        <div class="header-items">
+            <div class="header-images">
+
+            </div>
+            <h2>
+                Events
+            </h2>
+            <h3>
+                Lorem ipsum
+            </h3>
+        </div>
+        <div class="header-items">
+            <div class="header-images">
+
+            </div>
+            <h2>
+                Jobs
+            </h2>
+            <h3>
+                Lorem ipsum
+            </h3>
+        </div>
+        <div class="header-items">
+            <div class="header-images">
+
+            </div>
+            <h2>
+                Membership
+            </h2>
+            <h3>
+                Lorem ipsum
+            </h3>
+        </div>
+        <div class="header-topics">
+
+        </div>
+    </header>
+
+    <article class="container-top">
+        <div class="more-articles">
+            <p>Don't miss these latest<br>articles.</p>
+        </div>
+        <div class="change-colors"></div>
+        <div class="top-box">
```

Comment:
- [ ] `<div>`s of the .top-box class and other divs that represent articles should use the `<article>` tag. Make sure you check all the `div`s that have self-contained element representing an article and use the semantic tag for them.